algorithms/kernel/k_nearest_neighbors/bf_knn_impl.i

```diff
@@ -94,107 +72,203 @@ public:
             DAAL_CHECK_MALLOC(trainLabel);
         }
 
-        daal::algorithms::internal::EuclideanDistances<FPType, cpu> euclDist(
-            *testTable, *trainTable, !(resultsToCompute & computeDistances) && !(voteWeights & VoteWeights::voteDistance));
+        daal::algorithms::internal::EuclideanDistances<FPType, cpu> euclDist(*testTable, *trainTable, true);
```

Comment:
Why compute distances without sqrt (set squared=false)?

Reply:
Computed in another place